src/Management/src/Endpoint/Actuators/Health/HealthEndpointOptions.cs

```diff
@@ -8,7 +8,7 @@ namespace Steeltoe.Management.Endpoint.Actuators.Health;
 
 public sealed class HealthEndpointOptions : EndpointOptions
 {
-    public ShowDetails ShowDetails { get; set; }
+    public ShowDetails ShowDetails { get; set; } = ShowDetails.Never;
```

Comment:
I recommend reordering the enum members such that it defaults to Never, instead of explicitly setting it here.
Like this: `{ Never, WhenAuthorized, Always }`

The reason is that it's safer that way, in case there are more places (possibly in user code).

Reply:
Was it intentional to use `{ Never, Always, WhenAuthorized }` instead of the suggested `{ Never, WhenAuthorized, Always }`?

Reply:
Whoops, I didn't connect the dots that you were suggesting reordering all 3, I just swapped the first two so the default would change.